spacy/util.py

```diff
@@ -1736,3 +1737,40 @@ def all_equal(iterable):
     (or if the input is an empty sequence), False otherwise."""
     g = itertools.groupby(iterable)
     return next(g, True) and not next(g, False)
+
+
+def is_port_in_use(port, host="localhost"):
```

Comment:
- Let's add a docstring here and specify the argument types.
- Is `is_port_in_use()` called outside of `spacy/util.py`? If not (and we don't intend to), I'd suggest renaming this to `_is_port_in_use()`.

Reply:
Done.